packages/server/lib/cloud/exception.ts

```diff
@@ -1,17 +1,17 @@
 import _ from 'lodash'
-const Promise = require('bluebird')
-const pkg = require('@packages/root')
-const api = require('./api').default
-const user = require('./user')
-const system = require('../util/system')
-const { stripPath } = require('./strip_path')
+import Bluebird from 'bluebird'
+import pkg from '@packages/root'
+import api from './api'
```

Comment:
Since this file is imported in the test, I went ahead and finished converting it to use import statements to make is ESM compliant with vitest